index.html

```diff
@@ -172,5 +172,29 @@ <h3 class="about__links">LET'S CONNECT</h3>
         <hr>
     </div>
   </section>
+  <section class="contact_form">
+    <div class="contact__form">
+        <h2 class="contact__title">Contact me</h2>
+        <p class="contact__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="https://formspree.io/f/xgebklbg" method="post" class="form__list">
+        <ul class="form__list__ul">
+            <li>
+                <label for="name"></label>
+                <input class="input__li input__name" type="text" id="name" maxlength="30" placeholder="Enter your name" required>
+            </li>
+            <li>
+                <label for="email"></label>
+                <input class="input__li input__mail" type="email" id="email" placeholder="Enter your email" required>
+            </li>
+            <li>
+                <textarea class="input__li input__message" name="message" id="message" maxlength="500" placeholder="Write your message here" required></textarea>
+            </li>
+            <li>
+                <button class="submit__form" type="submit">Get in touch</button>
+            </li>
+        </ul>
```

Comment:
- [OPTIONAL] What do you think  🤔 about aligning these form fields using flexbox? It would make the form responsive.

Reply:
I aligned them with flexbox.

Thanks @wayungi